analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -115,10 +116,18 @@ object PythonVersion : CommandLineTool, Logging {
         }
 }
 
+private const val OPTION_OPERATING_SYSTEM = "operatingSystem"
+private const val OPTION_OPERATING_SYSTEM_DEFAULT = "linux"
+private val OPERATING_SYSTEMS = listOf("linux", "mac", "windows")
```

Comment:
Consider renaming `OPTION_OPERATING_SYSTEM_DEFAULT` to `OPTION_OPERATING_SYSTEM_LINUX`
to enable re-use in line 121 and maybe also next to @sschuberth 's comment below.

Reply:
I'd rather emphasize that this is the default value of the option than what the value is.